packages/sdk/src/types.ts

```diff
@@ -235,3 +235,5 @@ export type Expand<T> = T extends infer O ? { [K in keyof O]: O[K] } : never
 export * from './merkle/types'
 
 export type AddressInputType = 'DEPLOYER' | 'SOUND_EDITION' | 'MINTER' | 'FUNDING_RECIPIENT' | 'METADATA_MODULE'
+
+export type { ContractErrorName, ContractErrorSigHashToName } from './utils/constants'
```

Comment:
```suggestion
export { ContractErrorName, ContractErrorSigHashToName } from './utils/constants'
```

We should export the whole enums, not just the types

Reply:
https://github.com/soundxyz/sdk/pull/175/commits/07b2f8ac20133e6ee612ae3de7ef4ff296e28c51